data/json/techniques.json

```diff
@@ -2832,7 +2832,7 @@
     "type": "technique",
     "id": "tec_taichi_precise",
     "name": "Double Palm Strike",
-    "messages": [ "You double-handed palm strike %s", "<npcname> double-handed palm strikes %s" ],
+    "messages": [ "You double-handed palm strike %s!", "<npcname> double-handed palm strikes %s!" ],
```

Comment:
`"Your double-handed palm strikes %s!` and `<npcname>'s double-handed palm strikes %s!`